app/views/opportunities/_questions.erb

```diff
@@ -10,9 +10,14 @@
 <% end %>
 
 <% if policy(@opportunity).ask_question? %>
-  <div class='opportunity_section'>
+  <div class='opportunity_section' id="ask_question">
```

Comment:
Ah, thanks for calling that out! Just amended.

On Mon, May 2, 2016 at 1:05 PM, Adam Becker <notifications@github.com>
wrote:

> In app/views/opportunities/_questions.erb
> <https://github.com/dobtco/dispatch/pull/49#discussion_r61794849>:
>
> > @@ -10,9 +10,14 @@
> >  <% end %>
> >
> >  <% if policy(@opportunity).ask_question? %>
> > -  <div class='opportunity_section'>
> > +  <div class='opportunity_section' id="ask_question">
>
> But we're styling it:
> https://github.com/dobtco/dispatch/blob/ecb1e6d7717eed4859f55aa34875a549b8fe8b6f/themes/dvl-core/assets/stylesheets/theme/components/questions.scss#L8
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/dobtco/dispatch/pull/49/files/3f4f74be4e14682602fc7d8c376475e648c501e5#r61794849>
>